### PR TITLE
24140 new ens initial project solution setup changes

### PR DESCRIPTION
**Pull request by Ashish15251**
ENS Project Setup with Functional & Unit Test Project.

### CODE REVIEW

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/UKHO.ExternalNotificationService.API.FunctionalTests.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/UKHO.ExternalNotificationService.API.FunctionalTests.csproj:4]: Please change the Target Framework to net6.0

**Review by barrie-cooper — COMMENTED** [review]
One tiny change plus please move `.editorconfig` back to the root of the repo.

**Review by usha13529 — APPROVED** [review]
Changes looks good to me. Happy to approve this PR.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks for changes

**Comment by barrie-cooper** [comment]
Waiting on "PBI 24141 ENS - Azure DevOps initial CI Build" for merge
https://dev.azure.com/ukhydro/External%20Notification%20Service/_workitems/edit/24141

**Review by barrie-cooper — APPROVED** [review]

**Pull request merged**